cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,613 @@
+        name = TextInput(name="name")
+        name_help_text = Text(locator='.//div[contains(@id,"name")]/span')
+        description = TextInput(name="description")
+        select_vm = RadioGroup('.//div[contains(@id,"vm_choice_radio")]')
+
+        @property
+        def is_displayed(self):
+            return self.infra_map.is_displayed and self.name.is_displayed
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button("Import")
+        importcsv_btn = Button("Import CSV")
+        hidden_field = HiddenFileInput(locator='.//input[contains(@accept,".csv")]')
+
+        table = Table(
+            './/div[contains(@class, "container-fluid")]/table',
+            column_widgets={
+                "Select": Checkbox(locator=".//input"),
+                1: Text('.//span/button[contains(@class,"btn btn-link")]'),
+            },
+        )
+        filter_by_dropdown = SelectorDropdown("id", "filterFieldTypeMenu_vms_step")
+        search_box = TextInput(
+            locator='.//div[contains(@class, "modal-content")]'
+            '//div[contains(@class,"input-group")]/input'
+        )
+        clear_filters = Text(".//a[text()='Clear All Filters']")
+        error_text = Text(
+            './/h3[contains(@class,"blank-slate-pf-main-action") and '
+            'contains(text(), "Error:")]'
+        )
+        error_icon = Text(locator='.//span[contains(@class, "pficon-error-circle-o")]')
+        popover_text = Text(locator='.//div[contains(@class, "popover-content")]')
+
+        def after_fill(self, was_change):
+            self.browser.send_keys(Keys.ENTER, self.search_box)
+
+        @property
+        def is_displayed(self):
+            return self.table.is_displayed and (
+                len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0
+            )
+
+        def filter(self, type, name):
+            try:
+                self.filter_by_dropdown.item_select(type)
+            except NoSuchElementException:
+                logger.info("{} not present in filter dropdown!".format(type))
+            self.search_box.fill(name)
+            self.after_fill(was_change=True)
+
+        def select_by_name(self, vm_name):
+            self.filter("VM Name", vm_name)
+            vms_selected = []
+            for row in self.table.rows():
+                if vm_name in row.read()["VM Name"]:
+                    row.select.fill(True)
+                    vms_selected.append(row.read()["VM Name"])
+            return vms_selected
+
+    @View.nested
+    class advanced(View):  # noqa
+        pre_playbook = BootstrapSelect("preMigrationPlaybook")
+        post_playbook = BootstrapSelect("postMigrationPlaybook")
+        pre_checkbox = Text(locator='.//input[contains(@id, "pre_migration_select_all")]')
+        post_checkbox = Text(locator='.//input[contains(@id, "post_migration_select_all")]')
+
+        @property
+        def is_displayed(self):
+            return self.pre_playbook.is_displayed
```

Comment:
I would update `is_displayed` here when the table widget is added

Reply:
Will address this in next PR .